presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaConnectorFactory.java

```diff
@@ -36,10 +37,19 @@
         implements ConnectorFactory
 {
     private final Optional<Supplier<Map<SchemaTableName, KafkaTopicDescription>>> tableDescriptionSupplier;
+    private final Module additionalModule;
 
     KafkaConnectorFactory(Optional<Supplier<Map<SchemaTableName, KafkaTopicDescription>>> tableDescriptionSupplier)
+    {
+        this(tableDescriptionSupplier, new KafkaConsumerModule());
+    }
+
+    KafkaConnectorFactory(
```

Comment:
Nit: I wouldn't wrap here, as it looks funny / different compared to the above. Keeping the wrapping the same would make it easy to see that this is simply adding another argument.